modules/ROOT/pages/index.adoc

```diff
@@ -2,6 +2,9 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:useu-regions: (US and EU Regions)
```

Comment:
Did we decide on "Regions"? I thought we were using "Clouds".

Reply:
deleting, not used